errors/errors.go

```diff
@@ -23,4 +23,6 @@ var (
 	TypeWhitespaceError   = "field contains invalid characters (e.g., spaces, line breaks)"
 	TypeValueError        = "object type field doesn't contain valid value"
 	VisibilityValueError  = "ovisibility field doesn't contain valid value"
```

Comment:
Notice I made a typo in this value in the visibility check PR, got carried over to here.

Reply:
I can fix both this and the profile issue